olm/prepare-community-operators-update.sh

```diff
@@ -75,11 +78,29 @@ do
   "${packageFolderPath}/${lastPackagePreReleaseVersion}/${packageName}.v${lastPackagePreReleaseVersion}.clusterserviceversion.yaml" \
   > "${folderToUpdate}/eclipse-che.v${lastPackagePreReleaseVersion}.clusterserviceversion.yaml"
 
+  echo
   echo "   - Copying the CRD file"
   cp "${packageFolderPath}/${lastPackagePreReleaseVersion}/${packageName}.crd.yaml" \
   "${folderToUpdate}/eclipse-che.crd.yaml"
+  echo
   echo "   - Updating the 'final' channel with new release in the package descriptor: ${destinationPackageFilePath}"
   sed -e "s/${lastPublishedPackageVersion}/${lastPackagePreReleaseVersion}/" "${destinationPackageFilePath}" > "${destinationPackageFilePath}.new"
   mv "${destinationPackageFilePath}.new" "${destinationPackageFilePath}"
+  echo
+  echo "   - Committing changes"
+  cd "${communityOperatorsLocalGitFolder}"
+  git add --all
+  git commit -m "Update eclipse-che operator for ${platform} to release ${lastPackagePreReleaseVersion}"
+  echo
+  echo "   - Pushing branch ${branch} to the 'che-incubator/community-operators' GitHub repository"
+  if [ -z "${GIT_USER}" ] || [ -z "${GIT_PASSWORD}" ]
+  then
+    echo "#### WARNING ####"
```

Comment:
This sounds more like a failure with `exit 1` than a warning, since the `git push` won't get done.

Might be good to also echo the git push command to the console after this error message (but before the `exit 1`), so that once GIT_USER and password are set, the runner of the script can run the correct `git push ...` command.

Reply:
Well, this warning is emitted inside the loop on the platforms (kuberetes and openshift).
So if I want to exit 1, I should push these 2 calls outside of the loop.
I considered them as warnings because the user might want to push later on.

Reply:
> Might be good to also echo the git push command to the console after this error message, so that once GIT_USER and password are set, the runner of the script can run the correct git push ... command.

Done in commit https://github.com/eclipse/che-operator/pull/60/commits/eab0e8c00c26043012b96b7306f5573996929cd8

Reply:
:+1: